presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcFileWriter.java

```diff
@@ -256,6 +257,10 @@ private static ObjectInspector getJavaObjectInspector(TypeInfo typeInfo)
     private static StorageType toStorageType(Type type)
     {
         Class<?> javaType = type.getJavaType();
+        if (type instanceof DecimalType) {
```

Comment:
Place this before the line above, since `javaType` is not used in this `if` block